### PR TITLE
Organization concept removed from UV

### PR DESCRIPTION
**Pull request by eea03**
Organization concept removed from UV; Pipeline can now be owned again only by an user; However, pipelines (and other entities) now have metadata providing detailed information about the user;
Interface for pipeline execution changed to correspond with these changes

Pull request https://github.com/UnifiedViews/Core/pull/382 depends on these changes

### CODE REVIEW

File: uv-dpu-api/src/main/java/eu/unifiedviews/dpu/DPUContext.java [hunk 1]
```diff
@@ -99,9 +108,9 @@ void sendMessage(MessageType type,
     String getPipelineOwner();
 
     /**
-     * Return the pipeline owner organization
+     * If pipeline owner has meta data about the real user, returns Id, otherwise returns null
      */
-    String getOrganization();
```
Comment (skrchnavy) [inline uv-dpu-api/src/main/java/eu/unifiedviews/dpu/DPUContext.java]: Please check if this was used in plugins or in core, if yes, then we have to consider if keep this method for backward compatibility

**Comment by eea03** [comment]
Actually, the only usage was in L-FilesToVirtuoso DPU to create users in Virtuoso (which is optional eDemo feature that should be removed shortly anyway); So I don't see any reason to keep this method;

**Comment by skrchnavy** [comment]
@skodape please confirm it was not used in any of your DPUs.
@eea03 please confirm that UnifiedViews/Plugins master branch doesn't use this method (accidentally)
@mkl-eea-sk please check ODN related DPUs (loaders to CKAN)

**Comment by skodape** [comment]
Best to my knowledge this function is not used in our DPUs.

**Comment by eea03** [comment]
Unfortunately due to wrong implementation, this method is called in master branch version of VirtuosoLoader DPU outside the optional eDemo code block. I had hoped this would never be called without the special eDemo configuration property so we could remove it but now it looks we have to keep this method if we want to provide compatibility with older versions of VirtuosoLoader DPU.

CKAN loaders do not use this method; Looks like this method was implemented only because of this one DPU so the question is if we want to provide backward compatibility for it; If so, I return the method into interface as deprecated

**Comment by mkl-eea-sk** [comment]
Just note:
removing method from Interface should trigger changing major version number in some pom.xml. I cannot see such change in pull-request diff.

virtuosoloader can be upgraded and released with new release of UV/devenv and UV/core (based on the devenv). I think, there is no need to maintain compatibility where old DPU should be runnable on newer Core, when we provide new DPU which is able to run on new Core.
Debate would make sense, when we are unwilling to provide new DPU as a replacement.

**Comment by eea03** [comment]
I am aware of the versions, I just haven't discussed it with @skrchnavy yet but version will be adapted for sure;
I agree, I don't see a reason to provide such compatibility when it is one DPU and special functionality which no one will miss;

**Comment by skrchnavy** [comment]
@mkl-eea-sk I agree that removing method breaks backward compatibility, but if method was not used we can take the risk that we remove it and all DPUs works. 
Adding method triggers minor version change.

So we have to discuss how to cope with this, changing major version is not good option (lot of changes in rest of the system and maintenance of old version of api etc.)

File: uv-dpu-api/src/main/java/eu/unifiedviews/dpu/DPUContext.java [hunk 2]
```diff
@@ -161,23 +165,25 @@ void sendMessage(MessageType type,
 
     /**
      * @deprecated Do not use, will be removed in future versions. Use {@link DPUContext#getDpuInstanceDirectory()} to store any files shared among executions.
-     * Return existing DPU shared directory specific for single user. It's
-     * shared among all the instances and executions for single user and certain
-     * DPU (jar-file). Be aware of concurrency access when using this directory.
-     *
+     *             Return existing DPU shared directory specific for single user. It's
+     *             shared among all the instances and executions for single user and certain
+     *             DPU (jar-file). Be aware of concurrency access when using this directory.
```
Comment (tomas-knap) [inline uv-dpu-api/src/main/java/eu/unifiedviews/dpu/DPUContext.java]: Please adjust the formatting based on the template we use.

**Comment by tomas-knap** [comment]
Absolutely correct would be to increase the major version. But to simplify things and since it was not used by anybody outside of EEA/CUNI (because nobody knows how organizations should be used), I would suggest to simply remove it.

**Comment by eea03** [comment]
About the formatting: this was actually done by Eclipse using the UnifiedViews template; I even tried to import the template again to be sure I haven't done any changes but result was the same; Problem for this particular formatting is that it handles it as it belongs to the deprecated tag; We should try to fix this in the template itself (if it is possible) because I can fix it manually but whoever else comes next with the UV template and formatting of all lines on save (like me), problem will be back

**Comment by tomas-knap** [comment]
@eea03 Ok, then please adjust the template and please send info to the developers that the template was adjusted.

**Comment by tomas-knap** [comment]
Summary from call with @skrchnavy : We will leave getOrganization() as deprecated, to be correct.

**Comment by skrchnavy** [comment]
@eea03 please correct formating manualy. then #46 shall be fixed for formatter issue

**Comment by eea03** [comment]
OK, as agreed, getOrganization() method was put back as deprecated and formatting was manually fixed for now;

**Comment by skrchnavy** [comment]
@eea03 you can finish feature

**Comment by eea03** [comment]
Pull request UnifiedViews/Core#382 in Core is not ready yet. As discussed with @skrchnavy, this pull request should wait until it is finished

**Comment by skrchnavy** [comment]
Reason for waiting with merge is missing implementation of API in backend in develop branch (even dummy is not there)

**Pull request merged**